tests/integration_tests/conftest.py

```diff
@@ -17,6 +18,9 @@
     start_session_from_configuration,
 )
 
+GLOBAL_LOGGER = logging.getLogger(__name__)
+GLOBAL_LOGGER.propagate = True # we must set this to see stdout in caplog
```

Comment:
Do we need to name and track `GLOBAL_LOGGER`? 

```suggestion
logging.getLogger().propagate = True # we must set this to see stdout in caplog
```